src/image-copy/azext_imagecopy/cli_utils.py

```diff
@@ -65,7 +65,8 @@ def prepare_cli_command(cmd, output_as_json=True, tags=None, subscription=None):
         full_cmd += ['--tags', EXTENSION_TAG_STRING]
 
         if tags is not None:
-            full_cmd += tags.split()
+            for k, v in tags.items():
```

Comment:
tags could be key only

Reply:
don't we have common tags utilities in azure-cli core repo. how about leverage it?

Reply:
The purpose of this piece of code is to transform parsed tags to raw parameter values.